typedoc.config.js

```diff
@@ -21,7 +19,7 @@ module.exports = {
     ["@cennznet/api", "(.*packages\/api\/.*|node_modules\/@cennznet\/api\/.*)", "packages/api/README.md"],
     ["@cennznet/types", "(.*packages\/types\/.*|node_modules\/@cennznet\/types\/.*)", "packages/types/README.md"],
     ["@cennznet/util", "(.*packages\/util\/.*|node_modules\/@cennznet\/util\/.*)", "packages/util/README.md"],
-    ["@cennznet/crml-generic-asset", "(.*packages\/crml-generic-asset\/.*|node_modules\/@cennznet\/crml-generic-asset\/.*)", "packages/crml-generic-asset/README.md"],
+    ["@cennznet/crml-cennzx-spot", "(.*packages\/crml-attestaation\/.*|node_modules\/@cennznet\/crml-attestaation\/.*)", "packages/crml-attestation/README.md"],
```

Comment:
"@cennznet/crml-cennzx-spot" -> "@cennznet/attestation" and attestaation -> attestation